lib/v2/keylol/radar.js

```diff
@@ -0,0 +1,18 @@
+module.exports = {
+    'keylol.com': {
+        _name: '其乐',
+        '.': [
+            {
+                title: '论坛',
+                docs: 'https://docs.rsshub.app/game.html#qi-le-lun-tan',
+                source: ['/:category', '/'],
+                target: (params, url) => {
+                    const url = new URL(url);
```

Comment:
```suggestion
                target: (params, url) => {
                    url = new URL(url);
```
Function parameters cannot be redeclared.

Reply:
🙇‍♂️